Gemfile_next.lock

```diff
@@ -320,11 +319,11 @@ GEM
       activerecord (>= 4.0, < 6.1)
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    paypal-sdk-core (0.2.10)
+    paypal-sdk-core (0.3.4)
       multi_json (~> 1.0)
       xml-simple
-    paypal-sdk-merchant (1.106.1)
-      paypal-sdk-core (~> 0.2.3)
+    paypal-sdk-merchant (1.117.2)
```

Comment:
This one as well. This is the version we have in Gemfile.lock but not in Gemfile_next.lock

Reply:
Should be resolved by commit f8e7b76caf50a53810bbbc3700d274be97bbcdf3